src/test/java/com/google/sps/testAppReader_returnsAllAppsFromDb.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
```

Comment:
File name should be name of the class and suffixed with Test. They should also belong in the relevant directory, i.e.

```
.../sps/service/AppReaderTest.java
```